### ISSUE TITLE
Add support for CalibratedClassifierCV from scikit-learn

### ISSUE THREAD

**Issue by rspadim** [BayesWitnesses/m2cgen#462]
https://scikit-learn.org/stable/modules/generated/sklearn.calibration.CalibratedClassifierCV.html

it seens to be relative easy (a sigmoid function + mean)

https://github.com/scikit-learn/scikit-learn/blob/9b033758e/sklearn/calibration.py#L388
https://github.com/scikit-learn/scikit-learn/blob/9b033758e/sklearn/calibration.py#L433

**Comment by StrikerRUS** [comment]
Added into feature-request hub #491.
